app/views/layouts/_header.html.slim

```diff
@@ -1,20 +1,22 @@
 div class="bg-black"
   div class="row"
     div class="title-bar" data-responsive-toggle="main-menu" data-hide-for="medium"
-      button class="menu-icon" type="button" data-toggle=""
+      button[class="menu-icon" type="button" data-toggle]
```

Comment:
You can use `button.menu-icon[type="button" data-toggle]`

Also, why do you need `type="button"`?